server/src/internalClusterTest/java/org/opensearch/indexing/IndexActionIT.java

```diff
@@ -53,11 +59,25 @@
 import java.util.concurrent.Executors;
 import java.util.concurrent.atomic.AtomicIntegerArray;
 
+import static org.opensearch.indices.IndicesService.CLUSTER_REPLICATION_TYPE_SETTING;
 import static org.hamcrest.Matchers.containsString;
 import static org.hamcrest.Matchers.equalTo;
 import static org.hamcrest.Matchers.lessThanOrEqualTo;
 
-public class IndexActionIT extends OpenSearchIntegTestCase {
+public class IndexActionIT extends ParameterizedStaticSettingsOpenSearchIntegTestCase {
+
+    public IndexActionIT(Settings settings) {
+        super(settings);
+    }
+
+    @ParametersFactory
+    public static Collection<Object[]> parameters() {
+        return Arrays.asList(
```

Comment:
Any chance this can be declared in a common class? Seems like all the tests are doing it

Reply:
Thanks for the review @kkmr! We considered pulling this method up into a common class. However, there are some test suites that also use other settings like concurrent search that would override it. In that case we'll still have to redeclare this setting as a parameter in tandem with those. 

As a follow up, we can refactor the settings into a final hash map in the base class to reduce this method to a one-liner.

Reply:
Let's go ahead and introduce the common definition of the parameter list and use it in this PR. We can do a subsequent refactor PR to change existing parameterized tests.